OpenEdXMobile/build.gradle

```diff
@@ -238,7 +239,9 @@ configurations {
 def config = new TaskHelper().loadConfig(project)
 
 // https://developers.google.com/android/guides/google-services-plugin
-if (config.get('FIREBASE_ENABLED')?: false) {
+def firebase = config.get('FIREBASE')
+def firebaseEnabled = firebase?.get('ENABLED')
+if (firebaseEnabled?: false) {
```

Comment:
@miankhalid @farhan Is this the correct way to handle the modified config?

Reply:
way is right, but should be more optimized, make sure it works even if developer has not defined any of the config in its configuration file. Do testing of all possible cases.

**Proposed code:**
```
def firebase = config.get('FIREBASE')
if (firebase != null && firebase.get('ENABLED')) {
    apply plugin: 'com.google.gms.google-services'
}

```

Reply:
💡   Thought came into my mind, for the clean code why don't we remove the check on this pluggin and define it irrespective of enability of firebase, because we are already defining it in project level gradle file.
https://github.com/edx/edx-app-android/blob/9f204f8d603f488420c1c84164393545121d7793/build.gradle#L9

thoughts?
cc: @miankhalid

Reply:
👍